### PR TITLE
Fix WiFiscan not working when not connected

### PR DESCRIPTION
**Pull request by unlogisch04**
also changed tab to 4 space, and a 1 line if to a 3 line if

### CODE REVIEW

**Comment by ImUrX** [comment]
you should have made the tab change in a different commit, diff is not readable qwq
And the [editorconfig](https://github.com/SlimeVR/SlimeVR-Tracker-ESP/blob/main/.editorconfig) specifies tab instead of space

**Comment by unlogisch04** [comment]
Ok i had to search a littlebit what should be.
Seems like to be tab the correct one for future:
https://discord.com/channels/817184208525983775/959133879682342962/1089772491486011465
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/pull/235

**Comment by unlogisch04** [comment]
Ok, i did not have "EditorConfig for VS Code" installed so the rules where not applied on my code

File: src/serial/serialcommands.cpp [hunk 1]
```diff
@@ -196,7 +201,10 @@ namespace SerialCommands {
 			}
 
 			// Restore conencting state
-			if (WiFi.status() != WL_CONNECTED) WiFi.begin();
+			if (WiFi.status() != WL_CONNECTED) {
+				WiFi.begin();
+			}
+
```
Comment (TheDevMinerTV) [inline src/serial/serialcommands.cpp:207]:
```suggestion
			}
```

File: src/serial/serialcommands.cpp [hunk 2]
```diff
@@ -196,7 +201,10 @@ namespace SerialCommands {
...
         }
```
Comment (TheDevMinerTV) [inline src/serial/serialcommands.cpp:208]:
```suggestion
			if (WiFi.status() != WL_CONNECTED) {
				WiFi.begin();
			}
        }
```

**Review by TheDevMinerTV — APPROVED** [review]
LGTM

**Review by TheDevMinerTV — APPROVED** [review]
LGTM

**Comment by TheDevMinerTV** [comment]
ignore that I reviewed twice 😱

**Pull request merged**